stock_by_warehouse/static/src/js/widget.js

```diff
@@ -6,27 +6,34 @@ var Widget = require('web.Widget');
 var form_common = require('web.AbstractField');
 var formats = require('web.field_utils');
 var fieldRegistry = require('web.field_registry');
+var {EventDispatcherMixin} = require('web.mixins');
 var QWeb = core.qweb;
+var _t = core._t;
 
-var Locations = Widget.extend({
-    init: function (info) {
+var Locations = Widget.extend(EventDispatcherMixin, {
+    init: function (parent, info) {
+        this._super(...arguments);
```

Comment:
apply?

Reply:
Here the `apply` is not need it, when put the apply it give the following error:
`Uncaught TypeError: CreateListFromArrayLike called on non-object`

These changes were supervised by @JVegaB :see_no_evil: